R/deployApp.R

```diff
@@ -51,6 +51,9 @@
 #' @param metadata Additional metadata fields to save with the deployment
 #'   record. These fields will be returned on subsequent calls to
 #'   \code{\link{deployments}}.
+#' @param update If \code{TRUE}, always update the previously-deployed app.
```

Comment:
If I'm reading this correctly then `update = FALSE` means "ask me if I want to update" rather than "don't update", which could be confusing. I think we should either rename the option or give it a tri-state value (`always`, `never`, `prompt`).